rgthree-comfy and ComfyUI_Comfyroll_CustomNodes: No such file or directory: '/usr/share/fonts/truetype'

### Bug Description

Thanks for packaging this! I checked it out yesterday and today (pre and post v0.7.0 PR merge) and I'm getting this error trying to use a workflow containing rgthree-comfy. Although the custom node package comes pre-installed, I'm prompted to install it when I open a workflow with one of the nodes. After pressing install and reloading the page, no changes occur. Within my comfyui.service logs, this seems to be the only error.

```
Jan 02 21:16:02 host comfy-ui[1545355]: FileNotFoundError: [Errno 2] No such file or directory: '/usr/share/fonts/truetype'
```

I'm unsure if this problems are related or two separate problems, but as they're both preinstalled, I cannot remove one to isolate the problem. 

### Steps to Reproduce

1. Add flake to system configuration
2. Start comfyui.service
3. Open workflow containing `Fast Groups Bypass` or `Any Switch` nodes
4. Press "Install All" when prompted
5. View log

### Expected Behavior

rgthree-comfy should be pre-installed and not prompt for installation.

### Error Logs

```shell
Jan 02 21:15:49 host comfy-ui[1545355]: Starting server
Jan 02 21:15:49 host comfy-ui[1545355]: To see the GUI go to: http://127.0.0.1:8188
Jan 02 21:16:02 host comfy-ui[1545355]: [DEPRECATION WARNING] Detected import of deprecated legacy API: /scripts/ui.js. This is likely caused by a custom node extension using outdated APIs. Please update>
Jan 02 21:16:02 host comfy-ui[1545355]: [DEPRECATION WARNING] Detected import of deprecated legacy API: /extensions/core/clipspace.js. This is likely caused by a custom node extension using outdated APIs>
Jan 02 21:16:02 host comfy-ui[1545355]: [ERROR] An error occurred while retrieving information for the 'CR Select Font' node.
Jan 02 21:16:02 host comfy-ui[1545355]: Traceback (most recent call last):
Jan 02 21:16:02 host comfy-ui[1545355]:   File "/nix/store/xlkmjxl1rv66578j6al022lr3my2gkmz-source-patched/server.py", line 690, in get_object_info
Jan 02 21:16:02 host comfy-ui[1545355]:     out[x] = node_info(x)
Jan 02 21:16:02 host comfy-ui[1545355]:              ^^^^^^^^^^^^
Jan 02 21:16:02 host comfy-ui[1545355]:   File "/nix/store/xlkmjxl1rv66578j6al022lr3my2gkmz-source-patched/server.py", line 654, in node_info
Jan 02 21:16:02 host comfy-ui[1545355]:     info['input'] = obj_class.INPUT_TYPES()
Jan 02 21:16:02 host comfy-ui[1545355]:                     ^^^^^^^^^^^^^^^^^^^^^^^
Jan 02 21:16:02 host comfy-ui[1545355]:   File "/var/lib/comfyui/custom_nodes/ComfyUI_Comfyroll_CustomNodes/nodes/nodes_graphics_text.py", line 467, in INPUT_TYPES
Jan 02 21:16:02 host comfy-ui[1545355]:     file_list = [f for f in os.listdir(font_dir) if os.path.isfile(os.path.join(font_dir, f)) and f.lower().endswith(".ttf")]
Jan 02 21:16:02 host comfy-ui[1545355]:                             ^^^^^^^^^^^^^^^^^^^^
Jan 02 21:16:02 host comfy-ui[1545355]: FileNotFoundError: [Errno 2] No such file or directory: '/usr/share/fonts/truetype'
Jan 02 21:16:02 host comfy-ui[1545355]: []
Jan 02 21:16:02 host comfy-ui[1545355]: []
Jan 02 21:16:03 host comfy-ui[1545355]: [ComfyUI-Manager] The ComfyRegistry cache update is still in progress, so an outdated cache is being used.
```

### Platform

Linux (x86_64)

### ComfyUI Version

v0.6.0 and v0.7.0

### Additional Context

Installed on NixOS 26.05 (Yarara) x86_64

--- a/README.md
+++ b/README.md
@@ -93,6 +93,18 @@ Runtime packages (mutable):   <data-directory>/.pip-packages/
 
 A default manager config is created on first run with sensible defaults for personal use (`security_level=normal`, `network_mode=personal_cloud`).
 
+## Known Issues
+
+### Custom Nodes Requiring Patching
+
+Some custom nodes have hardcoded paths that don't exist on NixOS. We automatically patch these at startup, but **a full service restart is required after installing them via Manager**.
+
+| Node Pack | Issue | Fix |
+|-----------|-------|-----|
+| Comfyroll Studio (`ComfyUI_Comfyroll_CustomNodes`) | Hardcoded `/usr/share/fonts/truetype` | Patched to use bundled fonts |
+
+**After installing these nodes:** Stop ComfyUI completely (Ctrl+C) and restart with `nix run` or `systemctl restart comfyui`. Manager's internal restart does not trigger the patching.
+
 ## Bundled Custom Nodes
 
 The following custom nodes are bundled and automatically linked on first run:

--- a/nix/packages.nix
+++ b/nix/packages.nix
@@ -13,6 +13,25 @@ let
   # Template input files (images, audio, etc. for workflow templates)
   templateInputs = import ./template-inputs.nix { inherit pkgs; };
 
+  # Bundled fonts for custom nodes that require font files
+  # This provides a pure Nix solution for nodes like ComfyUI_Comfyroll_CustomNodes
+  # that expect fonts at hardcoded paths like /usr/share/fonts/truetype
+  bundledFonts = pkgs.symlinkJoin {
+    name = "comfyui-fonts";
+    paths = [
+      pkgs.dejavu_fonts
+      pkgs.liberation_ttf
+      pkgs.noto-fonts
+      pkgs.roboto
+    ];
+    postBuild = ''
+      # Create a flat directory with all TTF files for easy access
+      mkdir -p $out/ttf
+      find $out/share/fonts -name "*.ttf" -exec ln -sf {} $out/ttf/ \; 2>/dev/null || true
+      find $out/share/fonts -name "*.TTF" -exec ln -sf {} $out/ttf/ \; 2>/dev/null || true
+    '';
+  };
+
   # Import custom nodes for bundling
   customNodes = import ./custom-nodes.nix {
     inherit
@@ -298,6 +317,32 @@ let
               done
             fi
 
+            # Create fonts directory with bundled fonts for custom nodes
+            # This provides fonts for nodes like ComfyUI_Comfyroll_CustomNodes that expect
+            # fonts at hardcoded paths like /usr/share/fonts/truetype (which doesn't exist on NixOS)
+            FONTS_DIR="$BASE_DIR/fonts"
+            mkdir -p "$FONTS_DIR"
+            # Symlink bundled TTF fonts (only if not already linked)
+            for font_file in "${bundledFonts}"/ttf/*.ttf "${bundledFonts}"/ttf/*.TTF; do
+              if [[ -e "$font_file" ]]; then
+                filename=$(basename "$font_file")
+                target="$FONTS_DIR/$filename"
+                if [[ ! -e "$target" ]]; then
+                  ln -sf "$font_file" "$target"
+                fi
+              fi
+            done
+
+            # Patch custom nodes that use hardcoded font paths for NixOS compatibility
+            # ComfyUI_Comfyroll_CustomNodes: patch /usr/share/fonts/truetype -> $BASE_DIR/fonts
+            COMFYROLL_FONT_FILE="$BASE_DIR/custom_nodes/ComfyUI_Comfyroll_CustomNodes/nodes/nodes_graphics_text.py"
+            if [[ -f "$COMFYROLL_FONT_FILE" ]]; then
+              if grep -q '"/usr/share/fonts/truetype"' "$COMFYROLL_FONT_FILE" 2>/dev/null; then
+                sed -i "s|\"/usr/share/fonts/truetype\"|\"$FONTS_DIR\"|g" "$COMFYROLL_FONT_FILE"
+                echo "Patched ComfyUI_Comfyroll_CustomNodes for NixOS font compatibility"
+              fi
+            fi
+
             # Link our bundled custom nodes
             # Remove stale directories if they exist but aren't symlinks
             for node_dir in "model_downloader" "ComfyUI-Impact-Pack" "rgthree-comfy" "ComfyUI-KJNodes" "ComfyUI-GGUF" "ComfyUI-LTXVideo" "ComfyUI-Florence2" "ComfyUI_bitsandbytes_NF4" "x-flux-comfyui" "ComfyUI-MMAudio" "PuLID_ComfyUI" "ComfyUI-WanVideoWrapper"; do
@@ -359,7 +404,8 @@ let
             fi
 
             # Create default ComfyUI-Manager config if it doesn't exist
-            MANAGER_CONFIG_DIR="$BASE_DIR/user/default/ComfyUI-Manager"
+            # Note: Manager moved config from user/default/ComfyUI-Manager to user/__manager
+            MANAGER_CONFIG_DIR="$BASE_DIR/user/__manager"
             MANAGER_CONFIG="$MANAGER_CONFIG_DIR/config.ini"
             if [[ ! -e "$MANAGER_CONFIG" ]]; then
               mkdir -p "$MANAGER_CONFIG_DIR"

